doc_source/tls.rst

```diff
@@ -0,0 +1,78 @@
+.. Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+#####################################
+Enforcing TLS Version 1.2 in |sdk-go|
+#####################################
+
+.. meta::
+   :description: Describes how to set the TLS version for the |sdk-go|.
+
+To add increased security when communicating with AWS services, you should configure your client to use TLS 1.2 or later.
+TLS 1.2 has been the default in Go since release 1.13 in September 2019.
```

Comment:
I don't think `TLS 1.2` is the default in Go. The [Go 1.13](https://golang.org/doc/go1.13#library) release didn't mention this change, and the current Go documentation states TLS 1.0 as the default minimum version.

[doc](https://golang.org/pkg/crypto/tls/#Config.MinVersion)
```go
    // MinVersion contains the minimum TLS version that is acceptable.
    // If zero, TLS 1.0 is currently taken as the minimum.
    MinVersion uint16 // Go 1.2
```

Go did add standard support for TLS 1.3 in Go 1.13. [Go 1.14](https://golang.org/doc/go1.14#library) did remove support for SSL 3.0 though.

Reply:
I've removed the second sentence.